src/Time/Tags.hpp

```diff
@@ -22,9 +22,18 @@
 #include "Time/StepChoosers/StepChooser.hpp"        // IWYU pragma: keep
 #include "Time/StepControllers/StepController.hpp"  // IWYU pragma: keep
 #include "Time/Time.hpp"
+#include "Time/TimeAndPrevious.hpp"
 #include "Time/TimeStepId.hpp"
 #include "Utilities/TMPL.hpp"
 
+/// \cond
+namespace evolution {
+namespace Tags {
+struct PreviousTriggerTime;
+}  // namespace Tags
+}  // namespace evolution
```

Comment:
Shouldn't clang-tidy have complained about namespace nesting here?  I don't care if the code is changed, but I'm worried CI might not be working.

Reply:
As far as I can tell CI only runs clang-tidy on changed cpp files (see `tools/ClangTidyHash.sh`).  But even if I add the hpp files to the list of modified files, only the cpp files are used...